masterlist.yaml

```
@@ -1562,9 +1562,20 @@ plugins:
         util: 'FO4Edit v4.0.3h (Hotfix 1)'
 
   - name: 'WorkshopFramework.esm'
-    url: [ 'https://www.nexusmods.com/fallout4/mods/35004/' ]
+    url:
+      - link: 'https://www.nexusmods.com/fallout4/mods/35004/'
+        name: 'Workshop Framework (Nexus Mods)'
+      - link: 'https://bethesda.net/en/mods/fallout4/mod-detail/4081586'
```

Bethesda.net URLs can have trailing slashes:

```suggestion
      - link: 'https://bethesda.net/en/mods/fallout4/mod-detail/4081586/'
```

---

Sorry, you told me that before. I will eventually learn. :-)

---

No worries!